MongoDBLink.m

```diff
@@ -124,6 +126,18 @@
   DatabaseConnection[host, port, JavaNew["com.mongodb.MongoClient", host, port]]
 ]
 
+OpenConnection[host_String, port_Integer, "", _, _]:= OpenConnection[host, port]
```

Comment:
What's the use case for this overload? Having a blank string seems to make it very specific, or is that standard for indicating anonymous authentication?